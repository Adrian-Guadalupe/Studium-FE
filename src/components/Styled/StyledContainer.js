import React from 'react';
import styled from 'styled-components';

const StyledContainer = ( props ) => {
  return ( <Container { ...props }>
    { props.children }
  </Container> );
};

const Container = styled.div`
margin-top: ${ props => props.navBarVis ? '75px' : 0 };
margin-bottom: ${ props => props.navBarVis ? '75px' : 0 };
min-height: 100%;
position: absolute;
width: 100%;
max-height: 100vh;
max-width: 700px;
display: flex;
justify-content: ${ props => props.justifyContent || 'center' };
overflow-y: auto;
`;

export default StyledContainer;
