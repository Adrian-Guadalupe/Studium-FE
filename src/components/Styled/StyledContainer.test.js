import React from 'react';
import renderer from 'react-test-renderer';

import StyledContainer from './StyledContainer';



test('StyledContainer render correctly', () => {
    const tree = renderer.create(<StyledContainer />).toJSON();
    expect(tree).toMatchSnapshot();
});