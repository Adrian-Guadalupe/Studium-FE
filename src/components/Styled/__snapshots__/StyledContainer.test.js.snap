// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledContainer render correctly 1`] = `
.c0 {
  margin: 0 auto;
  min-height: 100%;
  position: absolute;
  width: 100%;
  max-height: 100vh;
  max-width: 700px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  overflow-y: scroll;
}

.c0::-webkit-scrollbar {
  width: 0;
}

<div
  className="c0"
/>
`;
